source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,284 @@
+sensitivity. By default, text searches run without case sensitivity
+which means MongoDB matches documents regardless of whether the value is
+lowercase or uppercase. 
+
+To specify a case sensitive search, use the following snippet:
+
+.. code-block:: java
+
+   TextSearchOptions options = new TextSearchOptions().caseSensitive(true);
+   Bson filter = Filters.text("SomeText", options);
+
+The following sections show you how to perform the following types of
+text searches:
+
+- Search Text by a Term
+- Search Text by a Phrase
+- Search Text with Terms Excluded
+- Sort by Relevance
+
+For more information, see the following API documentation:
+
+- :java-core-api:`Filters.text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass the term as a string to the ``Filters.text()`` method to specify a
+term in your text search. 
+
+If you want to match multiple terms in your text index, separate your
+terms with spaces. When you pass the ``Filters.text()`` method to the
+``find()`` method, the text search performs a logical OR on each term
+and returns documents that contain any of the terms. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "Fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin caseSensitiveExample
+   :end-before: end caseSensitiveExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the phrase with escaped quotes to the ``Filters.text()`` method to
+specify a phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+phrase "Fate of the Furious". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a term prefixed with a minus sign ``-`` to the ``Filters.text()``
+method to specify each term you want to exclude in your text search. 
+
```

Comment:
I think it could be helpful to explain how exclusion works. For example, as a reader, I'm curious about the following situations:
1) Can I specify only an exclusion term without any inclusion terms?
2) If my text index includes multiple fields and one of those fields contains the exclusion term, does that mean it is not a match?